src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionContainerBase.cs

```diff
@@ -255,6 +255,7 @@ public async Task<bool> WriteAckableMessageAsync(ServiceMessage serviceMessage,
                 case AckStatus.NotFound:
                     return false;
                 case AckStatus.Timeout:
+                case AckStatus.InternalServerError:
```

Comment:
Why it's throwing TimeoutException?

Reply:
From customer's perspective it is the same as Timeout